### PR TITLE
Cleans up metastation.

### PR DESCRIPTION
**Pull request by praisenarsie**
Make the beer keg nuke a subtype rather than a varedit and cleans up some nearstation areas.

### CODE REVIEW

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/reagents/reagent_dispenser.dm [hunk 1]
```diff
@@ -184,3 +184,9 @@
 	icon_state = "vat"
 	anchored = TRUE
 	reagent_id = "cooking_oil"
+
+/obj/structure/reagent_dispensers/beerkeg/nuke
+	name = "Nanotrasen-brand nuclear fission explosive"
+	desc = "One of the more successful achievements of the Nanotrasen Corporate Warfare Division, their nuclear fission explosives are renowned for being cheap to produce and devastatingly effective. Signs explain that though this particular device has been decommissioned, every Nanotrasen station is equipped with an equivalent one, just in case. All Captains carefully guard the disk needed to detonate them - at least, the sign says they do. There seems to be a tap on the back."
```
[the remark this answers is not in the document]
Comment (ShizCalev) [reply code/modules/reagents/reagent_dispenser.dm]: Move `There seems to be a tap on the back.` to an examine message.

`All Captains carefully guard the disk needed to detonate them - at least, the sign says they do.`
-> `All Captains carefully guard the disk needed to detonate them... or at least the sign says they do.`

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Review by ShizCalev — CHANGES_REQUESTED** [review]
lemme open a PR on your branch to fix the key changes.

**Comment by ShizCalev** [comment]
@praisenarsie https://github.com/praisenarsie/tgstation/pull/3

**Comment by ExcessiveUseOfCobblestone** [comment]
could you add an adjacency check to the tap message for... well you get the idea 😉

**Pull request closed without merging**